npm/package.json

```diff
@@ -4,7 +4,7 @@
   "version": "0.0.0",
   "license": "MIT",
   "engines": {
-    "node": ">=16.17.0"
+    "node": ">=18.17.1"
```

Comment:
I don't think we should bump this here. This restricts anyone who uses our package from npm to require them to be on 18.17.1 or later.

Reply:
OK, will revert